sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientLiveTests.cs

```diff
@@ -608,24 +608,29 @@ public async Task ProcessorClientDetectsAnInvalidEventHubName(bool async)
         }
 
         /// <summary>
-        ///   Verifies that the <see cref="EventProcessorClient" /> can read a set of published events.
+        ///   Verifies that the <see cref="EventProcessorClient" /> detects an invalid
+        ///   consumer group when starting up.
         /// </summary>
         ///
         [Test]
         [TestCase(true)]
         [TestCase(false)]
+        [Ignore("Ignore until the July release is completed, as this test was faulty and relies on base class changes to pass after fixes.")]
```

Comment:
Tracked by #29616.

This test requires a fix made in the core Event Hubs package and has been temporarily ignored until the July releases are complete.